sklearnex/basic_statistics/basic_statistics.py

```diff
@@ -14,4 +14,86 @@
 # limitations under the License.
 # ==============================================================================
 
-from onedal.basic_statistics import BasicStatistics
+from daal4py.sklearn._utils import control_n_jobs, run_with_n_jobs
+from onedal._device_offload import support_usm_ndarray
+from onedal.basic_statistics import BasicStatistics as onedal_BasicStatistics
+
+
+@control_n_jobs
+class BasicStatistics:
+    """
+    Estimator for basic statistics.
+    Allows to compute basic statistics for provided data.
+    Parameters
+    ----------
+    result_options: string or list, default='all'
+        List of statistics to compute
+
+    Attributes (are existing only if corresponding result option exists)
+    ----------
+        min : ndarray of shape (n_features,)
+            Minimum of each feature over all samples.
+        max : ndarray of shape (n_features,)
+            Maximum of each feature over all samples.
+        sum : ndarray of shape (n_features,)
+            Sum of each feature over all samples.
+        mean : ndarray of shape (n_features,)
+            Mean of each feature over all samples.
+        variance : ndarray of shape (n_features,)
+            Variance of each feature over all samples.
+        variation : ndarray of shape (n_features,)
+            Variation of each feature over all samples.
+        sum_squares : ndarray of shape (n_features,)
+            Sum of squares for each feature over all samples.
+        standard_deviation : ndarray of shape (n_features,)
+            Standard deviation of each feature over all samples.
+        sum_squares_centered : ndarray of shape (n_features,)
+            Centered sum of squares for each feature over all samples.
+        second_order_raw_moment : ndarray of shape (n_features,)
+            Second order moment of each feature over all samples.
+    """
+
+    def __init__(self, result_options="all"):
+        self.options = result_options
+
+    _onedal_incremental_covariance = staticmethod(onedal_BasicStatistics)
```

Comment:
Is it really relevant? Foe me personally it looks more like an artifact produced by a copy/paste😅

Reply:
Another question - are we okay with using attributes starting with one underscore in this scope? Should it be accessible from outside?

Reply:
> Is it really relevant? Foe me personally it looks more like an artifact produced by a copy/paste😅

yes, it is. thanks for pointing out.

> Another question - are we okay with using attributes starting with one underscore in this scope? Should it be accessible from outside?

do you mean it should start with two underscores? it is exactly what we discussed with @samir-nasibli and it looks like we have an agreement about naming "private" methods with one underscore to simplify development